### PR TITLE
3492: Correctly shorten name in drawer and notifications

### PR DESCRIPTION
**Pull request by Lakoja**
Fixes https://github.com/tuskyapp/Tusky/issues/3492

### CODE REVIEW

File: app/src/main/res/layout/item_status_notification.xml [hunk 1]
```diff
@@ -24,52 +24,59 @@
...
+                android:maxLines="1"
+                android:textColor="?android:textColorTertiary"
+                android:textSize="?attr/status_text_medium"
+                tools:text="\@Entenhausen" />
+
+        </LinearLayout>
 
         <TextView
             android:id="@+id/status_meta_info"
             android:layout_width="wrap_content"
```
Comment (Lakoja) [inline app/src/main/res/layout/item_status_notification.xml:71]: Have a width and stay visible.

However this does only work as second element in the LinearLayout thus the wrapping additional LinearLayout above is necessary.

**Comment by Lakoja** [comment]
Linter says: Nested weights are bad for performance

However that is not an error.
At least not until somebody finds another solution that works (this one here does).

**Comment by connyduck** [comment]
Here is how it works with ConstraintLayout: https://github.com/tuskyapp/Tusky/commit/b67806ee0e8ea7692da31b8385e07b7d247cddba

**Comment by Lakoja** [comment]
> Here is how it works with ConstraintLayout: [b67806e](https://github.com/tuskyapp/Tusky/commit/b67806ee0e8ea7692da31b8385e07b7d247cddba)

Ok, using that now, thank's.

**Review by connyduck — APPROVED** [review]

**Comment by fin-w** [comment]
Many thanks for your effort! It's appreciated :)

**Pull request merged**